### PR TITLE
Adding application

### PR DESCRIPTION
**Pull request by mikkjant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,4 @@
-# kubernetes-workshop
-This repository contains sources for Polar Squad [Kubernetes](https://kubernetes.io) workshop guide and material.
+# Apps
+This directory contains example applications for Istio workshop.
```
Comment (ernoaapa) [inline README.md]: Maybe for Kubernetes not Istio

File: apps/kubers/Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+APP_NAME = polarsquad/kubers
+
+all: v1
```
Comment (ernoaapa) [inline apps/kubers/Makefile]: Maybe rename the ‘v1’ to ‘build’ so it’s more describing and we don’t have multiple versions

**Review by ernoaapa — APPROVED** [review]
Minor tweaking, otherwise 👍

**Review by ernoaapa — COMMENTED** [review]
Maybe add also the steps to the circleCi configs so that the application get built and pushed always when get merged to master

**Comment by mikkjant** [comment]
CircleCi config now modified.

**Review by ernoaapa — APPROVED** [review]
👍

**Pull request merged**